### PR TITLE
fix: fix type radio in input component

### PR DESCRIPTION
**Pull request by cmcartaya**

### CODE REVIEW

File: src/components/Input/index.js [hunk 1]
```diff
@@ -1,167 +1,18 @@
...
-                </div>
-                <RenderIf isTrue={!!bottomHelpText}>
-                    <div className="rainbow-input_help">{bottomHelpText}</div>
-                </RenderIf>
-                <RenderIf isTrue={!!error}>
-                    <div id={this.getErrorMessageId()} className="rainbow-input_error">{error}</div>
-                </RenderIf>
-            </div>
-        );
+export function Input({ ...props }) {
```
Comment (LeandroTorresSicilia) [inline src/components/Input/index.js]: export function Input(props) {}

File: src/components/Input/index.js [hunk 2]
```diff
@@ -1,167 +1,18 @@
...
+    if (props.type === 'radio') {
```
Comment (LeandroTorresSicilia) [inline src/components/Input/index.js]: and then you can:
```
const { type } = props;
if (type === 'radio')
```

File: src/components/Input/inputBase.js [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+                </RenderIf>
+            </div>
+        );
+    }
+}
+
+InputBase.propTypes = {
+    /** Specifies the value of an input element. */
+    value: PropTypes.string,
+    /** The type of the input. This value defaults to text. */
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputBase.js]: remove this descriptions

File: src/components/Input/inputRadio.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+        this.errorMessageId = uniqueId('error-message');
+        this.inputRef = React.createRef();
+    }
+
+    getContainerClassNames() {
+        const { className, error, isCentered } = this.props;
+        return classnames(
+            'rainbow-input-radio_container',
+            { 'rainbow-input-radio--error': error },
+            { 'rainbow-input-radio_container--counter': isCentered }, className);
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio.js]: remove this

File: src/components/Input/inputRadio.js [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+        return classnames(
+            'rainbow-input-radio_label',
+            { 'rainbow-input-radio_label--disabled': disabled });
+    }
+
+    getBottomHelpTextClassName() {
+        const { isCentered } = this.props;
+        return classnames(
+            'rainbow-input-radio_help',
+            { 'rainbow-input-radio_help--counter': isCentered });
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio.js]: remove this

File: src/components/Input/inputRadio.js [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+    }
+
+    getErrorMessageClassName() {
+        const { isCentered } = this.props;
+        return classnames(
+            'rainbow-input-radio_error-message',
+            { 'rainbow-input-radio_error-message--counter': isCentered });
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio.js]: remove this

File: src/components/Input/inputRadio.js [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+            minLength,
+            pattern,
+            bottomHelpText,
+            required,
+            id,
+        } = this.props;
+
+        return (
+            <div id={id} className={this.getContainerClassNames()} style={style}>
+                    <span className="rainbow-input-radio_inner-container">
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio.js]: fix indentation

File: src/components/Input/readme.md [hunk 8]
```diff
@@ -8,6 +8,51 @@
...
+            disabled
+            label="Input Label" />
+    </div>
+
+
+##### input checkbox
+
+    <div className="rainbow-p-vertical_large rainbow-p-horizontal_xx-large rainbow-m-horizontal_xx-large">
+        <Input
+            type="checkbox"
```
Comment (LeandroTorresSicilia) [inline src/components/Input/readme.md]: remove this

**Comment by cmcartaya** [comment]
fix: #292

File: src/components/Input/inputBase/index.js [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+        return classnames('rainbow-input', {
+            'rainbow-input_bare': isBare,
+            'rainbow-input_counter': isCentered,
+        });
+    }
+
+    getInlineTextLabelId() {
+        const {
+            bottomHelpText,
+        } = this.props;
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputBase/index.js]: ```const { bottomHelpText } = this.props;```

File: src/components/Input/inputBase/styles.scss [hunk 10]
```diff
@@ -1,4 +1,4 @@
-/* stylelint-disable no-descending-specificity */
+/* stylelint-disable no-descending-specificity,max-line-length */
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputBase/styles.scss]: remove max-line-length

File: src/components/Input/inputRadio/index.js [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+        this.inlineTextLabelId = uniqueId('inline-text-label');
+        this.errorMessageId = uniqueId('error-message');
+        this.inputRef = React.createRef();
+    }
+
+    getContainerClassNames() {
+        const { className, error } = this.props;
+        return classnames(
+            'rainbow-input-radio_container',
+            { 'rainbow-input-radio--error': error }, className);
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/index.js]:
```
return classnames('rainbow-input-radio_container', {
    'rainbow-input-radio--error': error,
}, className);
```

File: src/components/Input/inputRadio/index.js [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+    }
+
+    getInlineTextLabelId() {
+        const {
+            bottomHelpText,
+        } = this.props;
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/index.js]:
```
const { bottomHelpText } = this.props;
```

File: src/components/Input/inputRadio/label.js [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+        label,
+        required,
+        inputId,
+        disabled,
+        id,
+    } = props;
+
+    const getLabelClassNames = () => classnames(
+        'rainbow-input-radio_label',
+        { 'rainbow-input-radio_label--disabled': disabled });
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/label.js]:
```
const getLabelClassNames = () => classnames('rainbow-input-radio_label', {
    'rainbow-input-radio_label--disabled': disabled,
});
```

File: src/components/Input/inputRadio/label.js [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+
+    return (
+        <RenderIf isTrue={!!label}>
+            <label className="rainbow-input-radio_label-container" htmlFor={inputId} id={id}>
+                <span className="rainbow-input-radio_faux" />
+                    <RequiredAsterisk required={required} />
+                <span className={getLabelClassNames()}>{label}</span>
+            </label>
+    </RenderIf>
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/label.js]: indentation

File: src/components/Input/inputRadio/index.js [hunk 15]
```diff
@@ -0,0 +1,155 @@
...
+            onChange,
+            label,
+            error,
+            disabled,
+            tabIndex,
+            onFocus,
+            onBlur,
+            onClick,
+            bottomHelpText,
+            required,
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/index.js]: remove required

File: src/components/Input/inputRadio/index.js [hunk 16]
```diff
@@ -0,0 +1,155 @@
...
+                        onClick={onClick}
+                        disabled={disabled}
+                        required={required}
+                        aria-labelledby={this.getInlineTextLabelId()}
+                        aria-describedby={this.getErrorMessageId()}
+                        ref={this.inputRef} />
+                    <Label
+                        disabled={disabled}
+                        label={label}
+                        required={required}
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/index.js]: same ^^

File: src/components/Input/inputRadio/index.js [hunk 17]
```diff
@@ -0,0 +1,155 @@
...
+
+InputRadio.propTypes = {
+    value: PropTypes.string,
+    label: PropTypes.oneOfType([
+        PropTypes.string, PropTypes.node,
+    ]),
+    bottomHelpText: PropTypes.oneOfType([
+        PropTypes.string, PropTypes.node,
+    ]),
+    required: PropTypes.bool,
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/index.js]: same ^^

File: src/components/Input/inputRadio/index.js [hunk 18]
```diff
@@ -0,0 +1,155 @@
...
+    className: PropTypes.string,
+    style: PropTypes.object,
+    id: PropTypes.string,
+};
+
+InputRadio.defaultProps = {
+    value: undefined,
+    label: null,
+    bottomHelpText: null,
+    required: false,
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/index.js]: same ^^

File: src/components/Input/inputRadio/label.js [hunk 19]
```diff
@@ -0,0 +1,42 @@
...
+
+    const getLabelClassNames = () => classnames('rainbow-input-radio_label', {
+        'rainbow-input-radio_label--disabled': disabled,
+    });
+
+    return (
+        <RenderIf isTrue={!!label}>
+            <label className="rainbow-input-radio_label-container" htmlFor={inputId} id={id}>
+                <span className="rainbow-input-radio_faux" />
+                    <RequiredAsterisk required={required} />
```
Comment (LeandroTorresSicilia) [inline src/components/Input/inputRadio/label.js]: remove this

**Pull request merged**